lib/puppet/provider/templates/webadministration/_getwebsites.ps1.erb

```diff
@@ -7,6 +7,19 @@ Get-WebSite | % {
     $preloadenabled = [string](Get-ItemProperty -Path "IIS:\Sites\$($name)" -Name 'applicationDefaults.preloadEnabled' -ErrorAction 'Continue').Value
   }
 
+  $authenticationTypes = @(
+    'anonymous',
+    'basic',
+    'clientCertificateMapping',
+    'digest',
+    'iisClientCertificateMapping',
+    'windows'
+  )
+  $authenticationTypes | % { $authenticationinfo = @{} } {
```

Comment:
Is the `$authenticationinfo = @{}` in the wrong place (Should be before the loop) or is this shorthand for;

ForEach-Object -Begin { ... } -Process { ... } ?

Reply:
Shorthand for begin process.